samples/tfm/tfm_psa_test/src/main.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2012-2014 Wind River Systems, Inc.
```

Comment:
Is this correct? If yes, can we have it in the sdk-nrf repo? Unsure.

Reply:
Fixed in #3728